electrum/gui/qt/__init__.py

```diff
@@ -105,6 +106,18 @@ def eventFilter(self, obj, event):
         return False
 
 
+class ScreenshotProtectionEventFilter(QObject):
+    def __init__(self):
+        super().__init__()
+
+    def eventFilter(self, obj, event):
+        if event.type() == QtCore.QEvent.Type.Show \
+                and isinstance(obj, QWidget) \
+                and obj.isWindow():
+                set_windows_os_screenshot_protection_drm_flag(obj)
```

Comment:
formatting: try to avoid having the same indentation level for both the condition and the inner code block, unless there is another line between. It is hard to see where the header ends and the body start.

E.g. this is more verbose but much more readable IMO:
```suggestion
        if (
            event.type() == QtCore.QEvent.Type.Show
            and isinstance(obj, QWidget)
            and obj.isWindow()
        ):
            set_windows_os_screenshot_protection_drm_flag(obj)
```

Reply:
Ok, changed it to your version: https://github.com/spesmilo/electrum/pull/9898/commits/ca9c4777d83381050dd426b7a10785d66b735426